public/data/ghorbani-mohammad.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Mohammad Ghorbani",
+    "type": "personal",
+    "bio": "Python Developer",
+    "avatar": "https://github.com/ghorbani-mohammad.png",
+    "links": [
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/tech_with_mike",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Programming",
+        "date": "December 215",
```

Comment:
```suggestion
        "date": "December 2015",
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Thank you for your consideration!